src/pages/HomePage/HeroSection.jsx

```diff
@@ -15,33 +34,124 @@ function HeroSection() {
         pt: { xs: '0px', sm: '0px' },
       }}
     >
-    <Box
-      sx={{
-        width: '100%',
-        height: { xs: '300px', sm: '600px' },
-        position: 'relative',
-        background: 'url(https://via.placeholder.com/1200x800) center/cover no-repeat',
-        display: 'flex',
-        alignItems: 'center',
-        justifyContent: 'center',
-      }}
-    >
-      <Box sx={{ textAlign: 'center', px: 4 }}>
-        <Typography variant="h3" fontWeight="bold" gutterBottom sx={{ color: '#fff' }}>
-          Step into Style and Comfort
-        </Typography>
-        <Typography variant="h6" gutterBottom sx={{ color: '#fff' }}>
-          Discover a wide range of trendy and comfortable shoes that make a statement.
-        </Typography>
-        <Button variant="contained" color="secondary" size="large">
-          Shop Now
-        </Button>
-      </Box>
-    </Box>
-      <Box mt={{ xs: 2, sm: 4, md: 0 }}>
+      <Box mt={{ xs: 2, sm: 4 }}>
         <FeaturesSection />
       </Box>
-      <Box sx={{ bgcolor: 'black', pb: 1 }}>
+
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: { xs: 'column', sm: 'row' },
+          alignItems: 'center',
+          justifyContent: 'center',
+          gap: { xs: 2, sm: 3 }, // Add space between cards, more for larger screens.
+          width: '100%',
+          maxWidth: '1200px',
+          mx: 'auto', // Centers the container if it's below the maximum width.
+          mt: 5, // Add space on top for better visual separation.
+          mb: 5, // Add space at the bottom as well for symmetry.
+        }}
+      >
+        {/* Basic Plan (Left) */}
+        <Card sx={{
+          width: { xs: '90%', sm: '28%' },  // Reduced width for mobile for breathing space.
```

Comment:
What's up with these %? It makes it hard to be consistent - try making them all the same width and using a grid with a spacing of 4. Also make the actual card comp its own custom one and then have the text info be props so it can easily be reused